src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/PackageTask.groovy

```diff
@@ -21,18 +21,42 @@ class PackageTask extends AbstractTask {
 
     @TaskAction
     void packageServer() {
+        
         def params = buildLibertyMap(project);
-        if (project.liberty.packageLiberty.archive != null && project.liberty.packageLiberty.archive.length() != 0) {
-            params.put('archive', new File(project.liberty.packageLiberty.archive))
+        def fileType = getPackageFileType(project.liberty.packageLiberty.include)
+        
+        def archive = project.liberty.packageLiberty.archive
+        
+        if (archive != null && archive.length() != 0) {
+            def archiveFile = new File(archive)
+            
+            if (archiveFile.exists() && archiveFile.isDirectory()) {
+                archiveFile = new File(archiveFile, project.getName() + fileType)
+            } 
+            params.put('archive', archiveFile)
+            logger.debug 'Packaging ' + archiveFile
+        } else {
+            // default output directory
+            def buildLibsDir = new File(project.getBuildDir(), 'libs')
```

Comment:
Is there any scenario that `${buildDir}/libs` doesn't exist what this task is executed?